index.html

```diff
@@ -1,42 +1,161 @@
+        <header>
+
+            <nav class="toolbar">
+                <div class="logo"><a href="#">My Logo</a></div>
+                <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+            </nav>
+         </header>
+             
+             <section class="first-page">
+          <div class="div-header">
+
+           <h1 class="header">I'm Kevin Glad to see you!</h1>  
+         <p class="about-par">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+               
+                    <a class="connect-page">LET'S CONNECT</a>
+                
+                <ul class="list">
+                    <li class="list-item"><img src="images/tweeter.png" alt="follow me on twitter" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/linkedin.png" alt="follow me on linkedin" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/medium.png" alt="follow me on medium" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/github.png" alt="follow me on github" width="20" height="20" ></li>
+                    <li class="list-item"><img src="images/angelist.png" alt="follow me on angellist" width="20" height="20" ></li>
+                </ul>
+                </div>
+        </section>  
+
+            <section class="work-section">
+               <div class="card">
+            <div class="section-img"><img src="images/snapshoot1.png"></div>
+            <div class="card-info">
+            <h2 class="work-header">Tonic</h2>
+            <ul class="descr-list">
+                <li class="descr-list-item1">CANOPY</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
+                <li class="descr-list-item"><img class="counter-img"src="images/Counter.png">2015</li>
+            </ul>
+            <p class="lang-descr">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="lang-list">
+                <li class="lang-list-descr">html</li>
+                <li  class="lang-list-descr">css</li>
+                <li  class="lang-list-descr">javaScript</li>
+            </ul>
+            <button class="btn-project">See Project</button>
+        </div>
+        </div>
+
+        <div class="card">
+            <div class="section-img"><img src="images/snapshoot2.png"></div>
+            <div class="card-info">
+            <h2 class="work-header">Multi-Post
+                Stories</h2>
+            <ul class="descr-list">
+                <li class="descr-list-item1">CANOPY</li>
+                <li class="descr-list-item"><img class="counter-img" src="images/Counter.png"> End Dev</li>
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design. I have noticed that
  - You have no margin between the headline section and the work section.
  - all your images in the work section are not showing
  
| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2023-01-12 17-36-43](https://user-images.githubusercontent.com/92303891/212139207-2f2e5a54-49b2-4275-b3ac-4ccd696eb099.png)| ![Screenshot from 2023-01-12 17-38-51](https://user-images.githubusercontent.com/92303891/212139611-d900a0a7-4c7d-454e-98ac-e7dfa057f1de.png)|